debezium-connector-db2/debezium-connector-db2.iml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you remove this one, we try not to commit IDE-specific configs.

Reply:
Agreed, removed.